app/locales/check.sh

```diff
@@ -9,7 +9,7 @@ echo "yarn i18n:extract"
 yarn i18n:extract
 
 echo "Checking for blank keys. Correct any that appear in this list:"
-! grep "\"\"" gps/app/locales/en.json
+! grep "\"\"" app/locales/en.json
```

Comment:
You must also update `pull.sh` and `push.sh` and the `i18n.yml` workflow file.

Reply:
good catch!